### PR TITLE
Fix #145184 www.lettuceclub.net

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

Fix #145184

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/Popups/sections/popups_specific.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@
 ! SECTION: Popups - Regular rules
 !
 bitrix24.ru##.helpdesk-notification-block
+lettuceclub.net###rewardModal
```
Comment (jellizaveta) [inline AnnoyancesFilter/Popups/sections/popups_specific.txt:11]: unfortunately, I can't reproduce this popup, could you send a screenshot with devtools open?

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -1254,6 +1254,7 @@ traveltoku.com##.g-single
 aumo.jp##.pure-ad-banner
 ||gpc-check.com/img/pr/
 lettuceclub.net##.l-contents--narrow
+lettuceclub.net###lcn-s_list_infeed1_gam
```
Comment (jellizaveta) [inline JapaneseFilter/sections/specific.txt:1257]: on which page is this element?

**Review by jellizaveta — CHANGES_REQUESTED** [review]
subscriptions:

<details><summary>Screenshot:</summary>

Screenshot 2023-03-15 at 13 51 29

</details><br/>

File: AnnoyancesFilter/Popups/sections/popups_specific.txt (hunk 1 above)
Comment (SKEIDs) [reply AnnoyancesFilter/Popups/sections/popups_specific.txt:11]: I can not reproduce now too.
I think the website changed.

Comment (jellizaveta) [reply AnnoyancesFilter/Popups/sections/popups_specific.txt:11]: well that happens, then you could delete these rules and add a rule for subscription

**Review by jellizaveta — APPROVED** [review]

**Pull request merged**